### DISCUSSION TITLE
Welcome to Scintillua Discussions!

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/scintillua#2] [General]
Feel free to discuss anything related to Scintillua here. The old mailing list is being retired. If you prefer not to use GitHub, you can still e-mail me personally (see README).

**Comment by dr-dba** [comment]
Hello, a bit of background - i am struggling about idea to find an ideal editor. That is the general motivation.
I have relatively strong grip on Lua programming, gradually get into it thru programming API of FAR manager.
Lets call FAR manager (http://farmanager.com/) "far-mgr" for brevity.
So coded for it, trying to make of far-mgr built-in programable editor something to go.
the far-mgr editor can be surprisingly very limited in some places, although really good at other
So, the experience not too much luck for me it, 
beside learnining Lua eventually, and other kinds of "selfdevelopment" probably :)

Thus and herafter, i look now towards TextAdept adoption, 
as for example, I need to know, how Scintillua can help starting with:
* certain line hiding (folding/colapsing)
* grepping abilities.

**As I understand, folding text is related to lexing, which is related to ScintiLua, 
which look attractive due my some knowledge about Lua programming.**
Together with that, i would be more than happy to get some of advice, 
and some general answers, before i am go into it too deep.. :)

Right now i know very little if at all of TextAdept and ScintiLua,
badly need some at least general advice.

I hope i explained myself, will be glad to continue any related discussion, 
from my side - i am sharing all my coding, so that all the community can get advance,
nothing to boast about, i am just trying to expalin myself, and my pursuits

**Comment by orbitalquark** [comment]
Hello! Scintillua is all about using lexers for syntax highlighting in editors (not just Textadept). It can also identify fold points to help editors know where to fold code. Feel free to open up a separate discussion with a question or topic, as this topic is just a general announcement.

**Reply by dr-dba** [reply]
i moved the message to another thread here, 
should I remove this original message?
